atlp-rwanda/atlp-devpulse-fn
# #152 Ft search job post

### PR Description
This PR Allows Applicant to search Job Posts

### Description of tasks that were expected to be completed
- Implement search Job post on for applicant side
- Allow applicant to select Job posts

### How has this been tested?
- Clone repo
- Checkout branch `ft-search-job-post`
- Make sure that you set the environment variables referring to `.env.example`
- Run `npm install`, then `npm run dev` to check if the app is running successfully

With deployed Link: [Preview](https://atlp-devpulse-fn-git-ft-sear-c7457f-devpulses-projects-62c56cf1.vercel.app/#/applicant/available-jobs)

### Credentials
Email: `test@applicant.com`
Password: `Bruest&#2500`

### Track PR (issue number & link)
[#152 ](https://github.com/atlp-rwanda/atlp-devpulse-fn/issues/152)

### Screenshots

1.Redirect to Job Posts page
![Screenshot (51)](https://github.com/user-attachments/assets/d902b7d7-a11d-4aa7-9218-2819567fb62a)


2. Select way you need to search 

![Screenshot (52)](https://github.com/user-attachments/assets/c4a45010-270b-489d-ab6e-63e1ee4c6e73)


3.Enter the search key you need then you will get Job posts related to it

![Screenshot (53)](https://github.com/user-attachments/assets/41ddc836-b7ee-4ede-97ee-b035a140cd44)
